sql/core/src/main/scala/org/apache/spark/sql/execution/command/SetCommand.scala

```diff
@@ -170,7 +170,11 @@ object SetCommand {
 case object ResetCommand extends RunnableCommand with IgnoreCachedData {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    sparkSession.sessionState.conf.clear()
```

Comment:
Could you update [the comment](https://github.com/apache/spark/pull/28003/files) for the new behaivour?

Reply:
Also, I think we need to clearly define the behaivour of `RESET` in [the doc](https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-aux-conf-mgmt-reset.md) like [hive](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Commands).